src/socketio.quiver.json

```diff
@@ -61,7 +61,7 @@
 
       "acceptUserInvitation": {
        "type": "method",
-        "value": ["string"]
+        "value": ["string", "string"]
```

Comment:
Did you consider making `inviteUserData` an arbitrary JSON object?  I think that would "just work", if you want the extra flexibility.